Fixed bug

arteevraina opened the pull request:
User was directed to landing page of add-in when no text was inserted inside` templateid` and `clauseid` text fields on clicking **ok** button which should not happen. Now, user will not be directed to landing page unless user enters  some text in ` templateid` and `clauseid` input fields.

Signed-off-by: Arteev Raina <arteevraina@gmail.com>

nik72619c commented on src/NewSmartClauseDialog.js:
```diff
@@ -46,13 +46,15 @@ class NewSmartClauseDialog extends Component {
-    const that = this;
-    const Office = window.Office;
-    const bindings = Office.context.document.bindings;
+    if(this.state.clauseId !== '' && this.state.templateId !== '') {
```
@arteevraina this check might not be able to cover all the use cases.
For example, if the user enters a space in the input then it might not work. you can try using something like :-
`if(this.state.clauseId.trim() !== '' && this.state.templateId.trim() !== '') {
 `

arteevraina replied:
Yes, I think it will make it correct

shakti97 commented on src/NewSmartClauseDialog.js:
```diff
@@ -93,6 +95,7 @@ class NewSmartClauseDialog extends Component {
               fullWidth
               value = {clauseId}
               onChange={this.handleClauseIdChange}
+              helperText="Please enter clauseid"
```
I think asterik are there to show that they are required text fields,
Giving helperText which make it permanently rendered you should do it like when the user clicks ok without fulfilling the required condition then only you should render "Please enter clauseId" in red color.

arteevraina replied:
yes that can be one option.. Thanks :)

arteevraina commented:
@DianaLease  I have made the text-field to highlight with red border when it is empty and whenever the user enters text in it it turns normal and when we hover the text-field it says `Please Fill out this field` because of required attribute.

arteevraina commented:
Now i have removed highlighted border and added red highlighted label text. As soon as user enters Text in the `Textfield` label color changes to normal color. This PR is ready for review @DianaLease
